doc/themes-breaking-changes.md

```diff
@@ -0,0 +1,73 @@
+---
+uid: Uno.Themes.BreakingChanges
+---
+
+# Overview
+
+The version 5.0.0 of Uno.Themes, introduced breaking changes on Converters, and Control Extensions. If you use one of those items on your project you will need to do some changes in order to keep using it.
+
+## Converter
+
+All Converters were moved to Uno.Themes library, and the new `namespace` is `Uno.Themes`.
```

Comment:
We should have a table of all converters with the old namespace and the new namespace like we do for other breaking change sections [here](https://github.com/unoplatform/Uno.Themes/blob/master/doc/material-migration.md#resources)